version: '3.7'
services:
  sync:
    image: wazoplatform/wait
    environment:
      TIMEOUT: '${INTEGRATION_TEST_TIMEOUT}'

  agentd:
    image: wazo-agentd-test
    environment:
      XIVO_UUID: 00000000-0000-0000-0000-0000000c4a7d
    volumes:
      - ../..:/usr/src/wazo-agentd:ro
      - ./etc/wazo-agentd/conf.d/50-default.yml:/etc/wazo-agentd/conf.d/50-default.yml:ro
      - ./key-file.yml:/tmp/key-file.yml:ro
      # - '${LOCAL_GIT_REPOS}/wazo-auth-client/wazo_auth_client:/opt/venv/lib/python3.7/site-packages/wazo_auth_client'
      # - '${LOCAL_GIT_REPOS}/xivo-bus/xivo_bus:/opt/venv/lib/python3.7/site-packages/xivo_bus'
      # - '${LOCAL_GIT_REPOS}/xivo-dao/xivo_dao:/opt/venv/lib/python3.7/site-packages/xivo_dao'
      # - '${LOCAL_GIT_REPOS}/xivo-lib-python/xivo:/opt/venv/lib/python3.7/site-packages/xivo'
    ports:
      - '9493'

  auth:
    image: wazoplatform/wazo-auth-mock
    ports:
      - '9497'

  amid:
    image: wazoplatform/wazo-amid-mock
    ports:
      - '9491'

  postgres:
    image: wazoplatform/wazo-confd-db-test
    ports:
      - "5432"

  rabbitmq:
    image: rabbitmq
    ports:
      - '5672'
    volumes:
      - type: tmpfs
        target: /var/lib/rabbitmq
